.github/workflows/docker-unified.yml

```diff
@@ -15,6 +15,10 @@ on:
   release:
     types: [published, edited]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
This means cancelling only those in progress for the same PR right?

Reply:
For the same branch